src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/checked-table-form/checked-table-form.component.scss

```diff
@@ -0,0 +1,4 @@
+label.datatable-permissions-scope-cell-label.custom-control-label,
+label.datatable-permissions-header-cell-label.custom-control-label {
+  margin-left: 6px;
+}
```

Comment:
we can remove this. because applying carbon to checkboxes must have taken care of the margin issue in the form, so i don't think this is needed anymore.
```suggestion

```

Reply:
Following code removed 
label.datatable-permissions-scope-cell-label.custom-control-label,
label.datatable-permissions-header-cell-label.custom-control-label {
  margin-left: 6px;
}